src/Domain.LinnApps/Requisitions/RequisitionService.cs

```diff
@@ -179,6 +179,34 @@ public async Task<RequisitionHeader> BookRequisition(int reqNumber, int? lineNum
             return req;
         }
 
+        public async Task<RequisitionHeader> AuthoriseRequisition(int reqNumber, User authorisedBy)
```

Comment:
i think it would be clearer to pass in employee id and privileges. This looks like it should be a real user

Reply:
I made the user object just to allow passing an Employee and their privileges as one object - feel free to delete and pass them seperately

Reply:
maybe leave it as it is for this PR to minimise conflicts for me and I'll split them all out next?